src/openzaak/utils/apidoc.py

```diff
@@ -34,4 +34,11 @@
 `user_id` en `user_representation` zijn nodig voor de audit trails. Het zijn
 vrije velden met als enige beperking dat de lengte maximaal de lengte van
 de overeenkomstige velden in de audit trail resources is (zie rest API spec).
+
+**Handige JWT links**
```

Comment:
I'dd suggest to leave here only a link to Authentication page at OZ RTD, and add other useful links there. Such way you will need to make changes only in RTD documentation in the future

Reply:
Just made it a single link at the bottom in possibly correct Dutch.